### PR TITLE
Fix the bug in video_base_model.py

### PR DESCRIPTION
**Pull request by My-Zhu**
When the metrics is None in the yaml files, old video_base_model.py will make the self.metric_results.items to zero but there is no this tensor actually. So this loop should be in the judgement of whether there are metric.

### CODE REVIEW

File: basicsr/models/video_base_model.py [hunk 1]
```diff
@@ -34,10 +34,9 @@ def dist_validation(self, dataloader, current_iter, tb_logger, save_img):
                     len(self.opt['val']['metrics']),
                     dtype=torch.float32,
                     device='cuda')
-
+            for _, tensor in self.metric_results.items():
```
Comment (xinntao) [inline basicsr/models/video_base_model.py:37]: 1. The program only enters L28 at the beginning of one experiment run, while L37 needs to executed when calling dist_validation in each validation phase. So this modification is not right.
2. I recommend adding an extra `if clause`  in previous version.

**Comment by xinntao** [comment]
Hi, @My-Zhu 
Thanks for contributing. 
I think that the current modification needs to be further improved. See the comments above. Thanks :-)

File: basicsr/models/video_base_model.py (hunk 1 above)
Comment (My-Zhu) [reply basicsr/models/video_base_model.py:37]: Thank you for your advice and I' ll check about that.

**Comment by My-Zhu** [comment]
> Hi, @My-Zhu
> Thanks for contributing.
> I think that the current modification needs to be further improved. See the comments above. Thanks :-)
Thanks for your wonderful work and I'll check about this modification.

**Comment by xinntao** [comment]
Hi, @My-Zhu 
If you are free, you may update this pull request. Thanks:-)

**Comment by My-Zhu** [comment]
Thank you for your reminding. I have updated the pull request according to your suggestion. :smiley:

**Comment by xinntao** [comment]
@My-Zhu Thanks!

**Pull request merged**